source/override-properties.d.ts

```diff
@@ -0,0 +1,24 @@
+import type {Merge} from './merge';
+
+/**
+Override existing property types in TOriginal with the types in TOverride.
```

Comment:
This should match the readme description.

Reply:
Done. I also rearranged the entry in the readme to be right after the merge types.